modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -2661,17 +2669,29 @@ public void removedService(
 	private class LiferayGraphQLFieldRetriever extends GraphQLFieldRetriever {
 
 		public GraphQLFieldDefinition getField(
-			boolean deprecated, Method method,
+			boolean deprecated, String graphQLNamespace, Method method,
 			ProcessingElementsContainer processingElementsContainer) {
 
 			GraphQLFieldDefinition.Builder graphQLFieldDefinitionBuilder =
 				_getGraphQLFieldDefinitionBuilder(
 					method, processingElementsContainer);
 
+			Class<?> clazz = method.getClass();
+
+			String canonicalName = clazz.getCanonicalName();
```

Comment:
I think it is easier here to use the existing pattern of pasing a `boolean mutation` to know if it is a mutation or a query